mne/label.py

```diff
@@ -2,7 +2,7 @@
 #          Martin Luessi <mluessi@nmr.mgh.harvard.edu>
 #          Denis Engemann <denis.engemann@gmail.com>
 #
-# License: BSD-3-Clause
+# License: BSD (3-clause)
```

Comment:
Revert, we settled on BSD-3-Clause in #9569

Reply:
Sorry incidental